post_installation_configuration/multi-architecture-configuration.adoc

```diff
@@ -26,6 +26,16 @@ include::modules/multi-architecture-modify-machine-set.adoc[leveloffset=+1]
 .Additional resources
 * xref:../machine_management/creating_machinesets/creating-machineset-azure.adoc[Creating a compute machine set on Azure] 
 
+== Creating a cluster with multi-architecture compute machines on AWS
+
+To create an AWS cluster with multi-architecture compute machines, you must first create a single-architecture AWS installer-provisioned cluster with the multi-architecture installer binary. For more information on AWS installations, refer to xref:../installing/installing_aws/installing-aws-customizations.adoc[Installing a cluster on AWS with customizations]. You can then add a ARM64 compute machine set to your AWS cluster and deploy your cluster with multi-architecture compute machines. 
+
```

Comment:
```and deploy your cluster with multi-architecture compute machines```

do we even need this? can be end the sentence here? because just adding the arm node makes it a multi-arch compute cluster.

Reply:
Yeah I agree, it was a bit difficult to not be repetitive since most of it was similar to Azure. But I do think removing it sounds better